.github/workflows/build_pr_documentation.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           cd ../../../../optimum-amd
           pip install .
 
-          pip install onnxruntime==1.14.0
+          pip install onnxruntime==1.14.0 torchvision
```

Comment:
maybe edit the setup.py as well if this is required

Reply:
done